### PR TITLE
hevc works, can toggle between h264 and hevc in alvr server

### PR DESCRIPTION
**Pull request by panthurrr**
quality is much nicer than h264 for me, hopefully higher res streaming soon !

### CODE REVIEW

File: ALVRClient/VideoHandler.swift [hunk 1]
```diff
@@ -20,32 +21,137 @@ struct VideoHandler {
...
+            } 
+        } else if (codec == HEVC_NAL_TYPE_VPS) {
+            let (vps, sps, pps, prefixSei, suffixSei) = extractParameterSets(from: initialNals)
+            
+            // Convert Data objects to arrays of UInt8 for each parameter set
+            let _ = vps != nil ? [UInt8](vps!) : nil
+            let _ = sps != nil ? [UInt8](sps!) : nil
+            let _ = pps != nil ? [UInt8](pps!) : nil
+            let _ = prefixSei != nil ? [UInt8](prefixSei!) : nil
+            let _ = suffixSei != nil ? [UInt8](suffixSei!) : nil
```
Comment (shadowfacts) [inline ALVRClient/VideoHandler.swift:51]: These can be removed

**Review by shadowfacts — CHANGES_REQUESTED** [review]

**Comment by invim** [comment]
Is there a video you can share of what you are seeing?

File: ALVRClient/VideoHandler.swift (hunk 1 above)
Comment (panthurrr) [reply ALVRClient/VideoHandler.swift]: removed

**Review by robbiet480 — APPROVED** [review]

**Comment by panthurrr** [comment]
> Is there a video you can share of what you are seeing?

Sure, GitHub doesn't let us upload anything too big but here's a few seconds with a controller in VR Chat.  Still a lot of room on improving the image quality though maybe someone with better wifi can get a good enough bitrate for native res.

https://github.com/alvr-org/alvr-visionos/assets/65619864/46049127-c063-4739-9910-5f70401c0878

**Comment by shinyquagsire23** [comment]
tested it merged w/ the latest changes in `master`, I did find one bug which is that `reduce color banding` breaks it. But IMO that's a PR of its own so I made an issue for it, #19

**Pull request merged**